src/Corax/IndexWriter.cs

```diff
@@ -242,10 +248,11 @@ void AddMaybeAvoidDuplicate(List<long> term, long entryId)
                     break;
 
                 case IndexEntryFieldType.Tuple:
-                    if (entryReader.Read(binding.FieldId, out Span<byte> valueInEntry) == false)
+                    if (entryReader.Read(binding.FieldId, out _,  out long lVal, out double dVal,out Span<byte> valueInEntry) == false)
                         break;
 
                     ExactInsert(valueInEntry);
+                    NumericInsert(lVal, dVal);
```

Comment:
`NumericInsert` for `TupleList ` is missing

Reply:
Please add that.